### DISCUSSION TITLE
Help to patch Async to binding-model of dxvk

### DISCUSSION THREAD

**Discussion by AussieNErd** [Sporif/dxvk-async#39] [Q&A] [answered]
@Sporif 

Hello 👋

I was wondering if you could help me by building an async patch of the latest commit to the dxvk binding-model branch.

https://github.com/doitsujin/dxvk/tree/binding-model

It will help with cpu bound dx11 games and I'd like to give it a test.

Thank you for your help🙂

Sean

**Comment by Sporif** [comment, marked as answer]
Here you go: [dxvk-async-1.10.1_102_g91de6454.tar.gz](https://github.com/Sporif/dxvk-async/files/8947881/dxvk-async-1.10.1_102_g91de6454.tar.gz)

**Comment by AussieNErd** [comment]
@Sporif 

Hello I wanted to thank you for doing that I really appreciate it. I was wondering if you could help me by doing an Async build the latest commit to the dxvk master branch.

Thank you 😊

**Reply by Sporif** [reply]
I think I will wait a while, the patch is breaking every few days it seems like, and It's too annoying to keep fixing it.

**Reply by Guy1524** [reply]
@Sporif In case you aren't aware, the latest commits to DXVK use the new graphics_pipeline_library extension to avoid stuttering, which may make the async branch obsolete.  Thanks for your work over the years rebasing this though!

**Reply by begin-theadventure** [reply]
In that case I'm waiting for a benchmark showing DXVK with graphics_pipeline_library vs DXVK with ~it +~ async.

**Reply by Sporif** [reply]
https://github.com/Sporif/dxvk-async/issues/44#issuecomment-1183676458
